src/components/ContinuationIn/SummaryBusinessHomeJurisdiction.vue

```diff
@@ -127,28 +127,29 @@
         </article>
 
         <!-- Extraprovincial Registration in B.C. -->
-        <template v-if="extraproRegistration">
-          <article class="section-container">
-            <v-row no-gutters>
-              <v-col
-                cols="12"
-                sm="3"
-                class="pr-4"
-              >
-                <label>Extraprovincial Registration in B.C.</label>
-              </v-col>
-              <v-col
-                cols="12"
-                sm="9"
-                class="pt-4 pt-sm-0"
-              >
-                <div id="extrapro-registration">
-                  {{ extraproRegistration }}
-                </div>
-              </v-col>
-            </v-row>
-          </article>
-        </template>
+        <article
+          v-if="isLookup"
+          class="section-container"
+        >
+          <v-row no-gutters>
+            <v-col
+              cols="12"
+              sm="3"
+              class="pr-4"
+            >
+              <label>Extraprovincial Registration in B.C.</label>
+            </v-col>
+            <v-col
+              cols="12"
+              sm="9"
+              class="pt-4 pt-sm-0"
+            >
+              <div id="extrapro-registration">
+                {{ extraproRegistration }}
+              </div>
+            </v-col>
+          </v-row>
+        </article>
```

Comment:
This is the bottom field here:

![image](https://github.com/bcgov/business-create-ui/assets/10002889/07987a78-5e60-4257-8d66-284421446c23)